src/App.tsx

```diff
@@ -145,15 +154,62 @@ function Footer() {
   );
 }
 
+function LanguageSelector(props: {
+  selectedLanguage: "en" | "de";
+  dispatch: React.Dispatch<Action>;
+}) {
+  const root = css`
+    display: flex;
+    width: fit-content;
+    align-self: center;
+    justify-self: flex-start;
+    grid-row: 1;
+    grid-column: 1/-1;
+    cursor: pointer;
+    user-select: none;
+    color: ${colors.accent};
+    align-items: center;
+    font-size: 24px;
+    padding: 4px 0;
+    opacity: 0.7;
+  `;
+
+  const text = css`
+    margin-right: 8px;
+  `;
+  const { i18n } = useTranslation();
+  return (
+    <div
+      className={root}
+      onClick={() => {
+        props.dispatch({ type: "toggle language" });
+        i18n.changeLanguage(props.selectedLanguage === "en" ? "de" : "en");
+      }}
+      role="button"
+    >
+      <div className={text}>
+        {i18n.language === "en" ? "English" : "Deutsch"}
+      </div>
+      {i18n.language === "en" ? (
+        <English width="40px" height="40px" />
+      ) : (
+        <German width="40px" height="40px" />
+      )}
```

Comment:
Can we remove the flags? I looks a little bit cleaner without them. Eventually I'll move all of this into a settings menu